flow.ts

```diff
@@ -132,19 +139,27 @@ export class AuthFlow {
     );
   }
 
-  private makeRefreshTokenRequest(code: string): Promise<void> {
+  private makeRefreshTokenRequest(code: string, codeVerifier?: string): Promise<void> {
```

Comment:
Given that you are always passing `codeVerifier` a better type would be `codeVerifier: string|undefined`